src/libuvc/libuvc.cpp

```diff
@@ -402,42 +407,173 @@ namespace librealsense
             }
             power_state get_power_state() const override { return _state; }
 
-            void init_xu(const extension_unit& xu) override {
-              // TODO : implement.
-            }
+            void init_xu(const extension_unit& xu) override {};
             bool set_xu(const extension_unit& xu, uint8_t control, const uint8_t* data, int size) override
             {
-              // TODO : implement.
+                uvc_set_ctrl(_device_handle, xu.unit, control, (void *)data, size);
```

Comment:
What happens when the xu fails, does it raise an exception ?

Reply:
I changed it to become symmetric with uvc_get_ctrl. uvc_get_ctrl let the underline call to treat the exception which makes more sense to support multiple backends.